subprojects/docs/src/docs/userguide/publishingIvy.adoc

```diff
@@ -191,14 +191,14 @@ Publications are published to repositories. The repositories to publish to are d
 </sample>
 ++++
 
-The DSL used to declare repositories for publishing is the same DSL that is used to declare repositories for dependencies (api:org.gradle.api.artifacts.dsl.RepositoryHandler[]). However, in the context of Ivy publication only the repositories created by the `ivy()` methods can be used as publication destinations. You cannot publish an `IvyPublication` to a Maven repository for example.
+The DSL used to declare repositories for publishing is the same DSL that is used to declare repositories for dependencies (api:org.gradle.api.artifacts.dsl.RepositoryHandler[]). However, in the context of Ivy publication only the repositories created by the `ivy()` methods can be used as publication destinations. You cannot publish an api:org.gradle.api.publish.ivy.IvyPublication[] to a Maven repository for example.
 
 [[publishing_ivy:publishing]]
 === Performing a publish
 
-The “`ivy-publish`” plugin automatically creates a api:org.gradle.api.publish.ivy.tasks.PublishToIvyRepository[] task for each api:org.gradle.api.publish.ivy.IvyPublication[] and api:org.gradle.api.artifacts.repositories.IvyArtifactRepository[] combination in the `publishing.publications` and `publishing.repositories` containers respectively.
+The Ivy Publish Plugin automatically creates a api:org.gradle.api.publish.ivy.tasks.PublishToIvyRepository[] task for each api:org.gradle.api.publish.ivy.IvyPublication[] and api:org.gradle.api.artifacts.repositories.IvyArtifactRepository[] combination in the `publishing.publications` and `publishing.repositories` containers respectively.
 
-The created task is named “`publish«_PUBNAME_»PublicationTo«_REPONAME_»Repository`”, which is “`publishIvyJavaPublicationToIvyRepository`” for this example. This task is of type api:org.gradle.api.publish.ivy.tasks.PublishToIvyRepository[].
+The created task is named `publish«_PUBNAME_»PublicationTo«_REPONAME_»Repository`, which is `publishIvyJavaPublicationToIvyRepository` for this example. This task is of type api:org.gradle.api.publish.ivy.tasks.PublishToIvyRepository[].
```

Comment:
I don't think the '«' character is standard. For example, we use \`clean__Task__\` or `\__sourceSet__RuntimeOnly\`. I guess this was done to avoid `publish__Publication__PublicationTo__Repository__Repository`. Gah, it's hard to write Asciidoc in a Markdown comment.